### ISSUE TITLE
Fedora gating test failure: 312 sdnotify : play kube - with policies

### ISSUE THREAD

**Issue by lsm5** [containers/podman#16246]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind bug

**Description**
```
not ok 312 sdnotify : play kube - with policies
# (from function `run_podman' in file ./helpers.bash, line 207,
#  in test file ./260-sdnotify.bats, line 341)
#   `run_podman container wait $container_a' failed
# # /usr/bin/podman rm -t 0 --all --force --ignore
# # /usr/bin/podman ps --all --external --format {{.ID}} {{.Names}}
# # /usr/bin/podman images --all --format {{.Repository}}:{{.Tag}} {{.ID}}
# quay.io/libpod/testimage:20220615 f26aa69bb3f3
# # /usr/bin/podman pull quay.io/libpod/fedora:31
# Trying to pull quay.io/libpod/fedora:31...
# Getting image source signatures
# Copying blob sha256:c28ace6b0c4ae099f6f81091731bdf41d9771d28bad96ae4a3507fe950560930
# Copying config sha256:a7a37f74ff864eec55891b64ad360d07020827486e30a92ea81d16459645b26a
# Writing manifest to image destination
# Storing signatures
# a7a37f74ff864eec55891b64ad360d07020827486e30a92ea81d16459645b26a
# # /usr/bin/podman container wait test_pod-a test_pod-b --condition=running
# Error: no container with name or ID "test_pod-a" found: no such container
# [ rc=125 ]
# # /usr/bin/podman ps -a
# CONTAINER ID  IMAGE       COMMAND     CREATED     STATUS      PORTS       NAMES
# # /usr/bin/podman container wait test_pod-a test_pod-b --condition=running
# Error: no container with name or ID "test_pod-a" found: no such container
# [ rc=125 ]
# # /usr/bin/podman ps -a
# CONTAINER ID  IMAGE                                    COMMAND     CREATED                 STATUS      PORTS       NAMES
# 0de3cb821f7e  localhost/podman-pause:4.3.0-1666271887              Less than a second ago  Created                 9cffc8377ee1-service
# 28bbe47814aa  localhost/podman-pause:4.3.0-1666271887              Less than a second ago  Created                 a43fed04129e-infra
# 68831fd785b8  quay.io/libpod/fedora:31                             Less than a second ago  Created                 test_pod-a
# # /usr/bin/podman container wait test_pod-a test_pod-b --condition=running
# -1
# -1
# # /usr/bin/podman container inspect test_pod-a test_pod-b 9cffc8377ee1-service --format {{.Config.SdNotifyMode}}
# container
# conmon
# ignore
# # /usr/bin/podman container inspect test_pod-a --format {{.Config.SdNotifySocket}}
# /var/tmp/-podman-notify-proxy.sock3209409983
# # /usr/bin/podman logs test_pod-a
# /run/notify/notify.sock
# READY
# # /usr/bin/podman exec test_pod-a /bin/touch /stop
# # /usr/bin/podman container inspect 9cffc8377ee1-service --format {{.State.ConmonPid}}
# 139810
# # /usr/bin/podman container wait test_pod-a
# timeout: sending signal TERM to command â€˜/usr/bin/podmanâ€™
# [ rc=124 (** EXPECTED 0 **) ]
# *** TIMED OUT ***
# # [teardown]
# # /usr/bin/podman pod rm -t 0 --all --force --ignore
# a43fed04129ef695d2e4fb2db4ed61e529e47c47585b8c3a83fce65b65f424c8
# # /usr/bin/podman rm -t 0 --all --force --ignore
# # /usr/bin/podman network prune --force
# podman-default-kube-network
# setup(): removing stray image localhost/podman-pause:4.3.0-1666271887
# setup(): removing stray image dd795a4eb195
# setup(): removing stray image quay.io/libpod/fedora:31
# setup(): removing stray image a7a37f74ff86
```

**Steps to reproduce the issue:**

1. Fedora gating tests

**Package info (e.g. output of `rpm -q podman` or `apt list podman` or `brew info podman`):**

```
4.3.0-2
```

**Comment by lsm5** [comment]
@edsantiago PTAL

**Comment by edsantiago** [comment]
This is #16076, which ~~was~~ should've been fixed a week ago in #16118. @vrothberg PTAL. @lsm5 please confirm that this is 4.3, not something old.

**Comment by lsm5** [comment]
> This is #16076, which ~was~ should've been fixed a week ago in #16118. @vrothberg PTAL. @lsm5 please confirm that this is 4.3, not something old.

It's the latest v4.3.0 on rawhide https://bodhi.fedoraproject.org/updates/FEDORA-2022-48156b03b5

**Comment by vrothberg** [comment]
> This is #16076, which ~was~ should've been fixed a week ago in #16118. @vrothberg PTAL. @lsm5 please confirm that this is 4.3, not something old.

The "fix" is part of v4.3.0 but apparently not sufficient.  Any chance we can restart gating tests?

Did we see the test flake in rawhide before?

**Comment by lsm5** [comment]
> The "fix" is part of v4.3.0 but apparently not sufficient. Any chance we can restart gating tests?
> 
> Did we see the test flake in rawhide before?

`before` meaning before v4.3.0 or on a prior test run with v4.3.0 ? Maybe @edsantiago remembers better.

**Comment by lsm5** [comment]
rerunning gating test now

**Comment by lsm5** [comment]
> In general. Any previous of version of Podman but this test.

ack, no recollection. I'll leave it to Ed.

**Comment by edsantiago** [comment]
There is no mechanism (push nor pull) for me to know of Fedora gating-test results, so I can neither monitor them nor save logs. I'm sorry, I have no idea what the history of this flake is. Since @vrothberg _just_ fixed it a week ago in `main`, I think the important consideration is that the fix in #16118 may not be sufficient.

**Comment by edsantiago** [comment]
Closing as dup of #16076.
